packages/main/test/pages/Switch.html

```diff
@@ -55,6 +60,15 @@ <h3>Graphical Switch</h3>
 		<ui5-switch design="Graphical" checked text-on="Yes" text-off="No" disabled></ui5-switch>
 	</div>
 
+	<h3>Switch in form, required</h3>
```

Comment:
We could simulate a real use case. For example "Accept company policies" for a form label and "Yes"/"No" for the switch.